service/dynamodb/expression/doc.go

```diff
@@ -0,0 +1,49 @@
+/*
+Package expression provides types and functions to create DynamoDB Expression
+strings, ExpressionAttributeNames maps, and ExpressionAttributeValues maps.
+
+Using the Package
+
+The package represents the various DynamoDB Expressions as builder structs named
+accordingly. For example, ConditionBuilder represents a DynamoDB Condition
+Expression, an UpdateBuilder represents a DynamoDB Update Expression, and so
+forth.
+The following example will show a sample ConditionExpression and how to build an
```

Comment:
future tense :)

`The following is an example using a condition builder to create the expression where DynamoDB record's field name "Artist" equals the value "No One You Know".